django-rgd-imagery/rgd_imagery/templates/rgd_imagery/imageset_detail.html

```diff
@@ -168,12 +101,41 @@
       });
     }
 
+    let imageLayer = null;
+
+    async function updateTiles() {
+      const image_id = Number(imageSelector.value).toString();
+      const band = Number(bandSelector.value).toString();
+
+      const tileinfo = await fetch(
+        `${host}/api/image_process/imagery/${image_id}/tiles`
+      ).then(response => response.json());
+
+      const params = geo.util.pixelCoordinateParams(
+        '#imageViewer', tileinfo.sizeX, tileinfo.sizeY, tileinfo.tileWidth, tileinfo.tileHeight);
+      params.layer.url = `${host}/api/image_process/imagery/${image_id}/tiles/{z}/{x}/{y}.png?band=${band}`;
+
+      const imageViewer = geo.map(params.map);
+
+      imageViewer.zoomRange({
+        // do not set a min limit so that bounds clamping determines min
+        min: -Infinity,
+        max: 12,
+      });
```

Comment:
The `params.map` has some settings for the zoom range (`min` and `max` should be set to allow full navigation within the image, `zoom` is the starting zoom level and is set to the min value).  If you want to be able to zoom further, subtracting from the `params.map.min` and adding to `params.map.max` before calling `map(params.map)` should be sufficient.